README.md

```diff
@@ -4,56 +4,53 @@
   <img src="./src/General_Grants_Program.png" style="width:1300px";>
 </p>
 
-**:loudspeaker: Important:** The Web3 Foundation General Grants Program focuses on technology grants which are meant to remain **private**, **paid out in fiat** or **larger than USD 30,000**. We ask teams to first consider applying via the [**Open Grants Program**](https://github.com/w3f/Open-Grants-Program), which supports grants of up to USD 30,000 and has **faster processing times**. If your project has a scope larger than that, we encourage you to split your work into multiple stages of USD 30,000 or less.
+**:loudspeaker: Important:** The Web3 Foundation _General Grants_ Program focuses on technology grants which teams wish to keep **private**, need to be **paid out in fiat** or are **larger than $30,000**. We ask teams to first consider applying via the [_Open Grants_ Program](https://github.com/w3f/Open-Grants-Program), which supports grants of up to $30,000 and has **faster processing times**. If your project has a scope larger than that, we encourage you to split your work into multiple smaller stages of $30,000 or less.
```

Comment:
Not really knowledgeable here, but `$` is ambiguous, no? `US$` maybe?

Reply:
It is, why is why I initially went with USD, but it looks a bit clunky and I think it is generally accepted and pretty clear that a simple $ refers to USD. I found [this](https://english.stackexchange.com/a/26414) discussion on the matter.